platforms/platforms_windows_test.go

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Since its `_test.go` do we need a build directive here for `windows`? Or is the test tool smart enough now for `*_windows_test.go` to maintain the platform?